handler/cantabular_metadata_export.go

```diff
@@ -118,6 +118,14 @@ func (h *CantabularMetadataExport) Handle(ctx context.Context, workerID int, msg
 		}
 	}
 
+	// for filter custom flag
+	// It feels like without this
+	// there would be nilpointers from it not being set in previous services
+	// but still being false
+	if filterOutput.Custom == nil {
+		falseFlag := false
+		filterOutput.Custom = &falseFlag
```

Comment:
Please remove this comment.. Either it will or it won't, you should be able to test whether it being nil will be a problem and where. If this is how you want to prevent it fine but having a comment that reads 'it feels like we need this but not really sure' is a huge smell.